packages/react-google-maps-api/src/components/drawing/Data.tsx

```diff
@@ -0,0 +1,160 @@
+/* global google */
```

Comment:
How many `<Data />` components can we render at the same time?

Reply:
I'm actually just seeing now that we don't export Data in index.ts. We need to add an example/docz page for it. Why do you ask? I don't see any constraint on the number of Data components that can be rendered